NitroxClient/Communication/NetworkingLayer/NitroxClient.cs

```diff
@@ -0,0 +1,27 @@
+using NitroxClient.Communication.Abstract;
+using NitroxModel.Logger;
+using NitroxModel.Networking;
+using NitroxModel.Packets;
+
+namespace NitroxClient.Communication.NetworkingLayer
+{
+    public abstract class NitroxClient : IClient
```

Comment:
This is a useless abstraction. An abstract class should only ever exist as a promise of default behavior. This class is nearly a functional equivalent of the IClient interface.